src/ngraph/runtime/cpu/cpu_debugger.cpp

```diff
@@ -19,12 +19,26 @@
 #include "ngraph/runtime/aligned_buffer.hpp"
 #include "ngraph/runtime/cpu/cpu_debugger.hpp"
 #include "ngraph/runtime/cpu/cpu_external_function.hpp"
+#include "ngraph/runtime/cpu/cpu_runtime_context.hpp"
 #include "ngraph/runtime/cpu/cpu_tensor_view.hpp"
 #include "ngraph/runtime/cpu/cpu_tracing.hpp"
 
 using namespace std;
 using namespace ngraph;
 
+void runtime::cpu::CPU_CountTracepoint::operator()(void** outputs, const std::string& name)
+{
+    if (m_count == 0)
+    {
+        return;
+    }
+    if (++m_iteration == m_count)
```

Comment:
Either do >= here, or add a clause for > and throw a runtime error if it happens.